cnx-contrib/Cargo.toml

```diff
@@ -26,3 +26,4 @@ byte-unit = "4.0.12"
 reqwest = { version = "0.11" }
 [target.'cfg(openbsd)'.dependencies]
 sioctl = { version = "0.0.1", optional = true}
+openssl = { version = "0.10", features = ["vendored"] }
```

Comment:
Any reason for adding `openssl` ?

Reply:
`openssl` is a dependency of `weathernoaa`. I added it explicitly to activate the feature `vendored`. Otherwise the pipeline fails.
This has nothing to do with the fix for the CPU calculation. So I am open for alternative solutions.